.github/workflows/validate_kong_image_trigger_via_label.yaml

```diff
@@ -30,6 +32,8 @@ jobs:
       kong_image_major_version: ${{ steps.semver_parser.outputs.major }}
       kong_image_minor_version: ${{ steps.semver_parser.outputs.minor }}
     steps:
+     - name: checkout repository
+       uses: actions/checkout@v4
```

Comment:
ditto

Reply:
Deleted. This job does not run `gh` command and does not refer to paths in the repo, so I guess it is OK not to checkout here.